docs/ai/conceptual/understanding-tokens.md

```diff
@@ -33,8 +33,6 @@ This text could be tokenized as:
 - a
 - cat
 
-The model will add only the first occurrence of the "a" token to its vocabulary. All other occurrences can be represented by that first token.
```

Comment:
Removed this line because it feels a little off-topic in this description and its covered later in the content.